commissioning.tex

```diff
@@ -1,77 +1,259 @@
+
+% DDF area summary 
+\subsubsection{SV Deep Drilling Fields}
+The four DDFs observed during the survey had variable completeness.
+Table \ref{tab:sv_ddf_summary}  lists the number of visits obtained during LSSTCam commissioning, including observations acquired during the  small-field  and SV surveys, within each Deep Drilling Field (DDF). 
+For each field, the first row gives the number of visits per band, while the second row reports the corresponding estimated coadded $5\sigma$ depths ($m_5$). 
+The total number of visits, the timespan over which observations were collected, and the median image quality, expressed in terms of the PSF at FWHM are also provided.
+\begin{table*}[htb]
+\centering
+\renewcommand{\arraystretch}{1.2}
+\begin{tabular}{lccccccccc}
+\toprule
+\textbf{Field} &
+\textbf{Timespan} &
+\textbf{Median FWHM} &
+\multicolumn{6}{c}{\textbf{Band}} &
+\textbf{Total} \\
+\cmidrule(lr){4-9}
+ & (days) & (arcsec) & $u$ & $g$ & $r$ & $i$ & $z$ & $y$ & \textbf{N visits} \\
+\midrule
+XMM\_LSS  & 10 & 1.47 & 30 & — & — & — & — & — & 30 \\
+           &    &      & 25.3 & — & — & — & — & — &  \\
+\addlinespace
+EDFS\_A   & 55 & 1.32 & — & 20 & 21 & 28 & 18 & — & 87 \\
+           &    &      & — & 25.3 & 25.2 & 25.0 & 24.1 & — &  \\
+\addlinespace
+EDFS\_B   & 55 & 1.37 & — & 23 & 21 & 29 & 18 & — & 91 \\
+           &    &      & — & 25.4 & 25.1 & 25.0 & 24.1 & — &  \\
+\addlinespace
+ECDFS     & 57 & 1.51 & — & 36 & 39 & 41 & 43 & — & 159 \\
+                 &      &         & — & 25.8 & 25.5 & 25.3 & 24.7 & — &  \\
+\addlinespace
+ELAISS1   & 84 & 1.40 & 39 & 101 & 103 & 168 & 112 & 16 & 539 \\
+                 &    &      25.6 & 26.6&  26.1 &  26.2 &  25.1 & 22.7 & \\        
+\addlinespace
+COSMOS    & 15 & 1.22 & 100 & 82 & 166 & 139 & 111 & 66 & 664 \\
+           &    &      & 26.0 & 26.8 & 26.3 & 26.1 & 25.3 & 23.9 &  \\
+\bottomrule
+	
+\end{tabular}
+\caption{Summary of Deep Drilling Field observations from LSSTCam on-sky commissioning data, including SV and small-field campaigns. For each field, the first row lists the number of visits per band and total, and the second row lists the corresponding coadded $5\sigma$ depths ($m_5$). The observation timespan and median image FWHM are also indicated.}
+\label{tab:sv_ddf_summary}
+\end{table*}
+
+The SV surveys occurred relatively early in the observing season for all Deep Drilling Fields, and COSMOS was entirely unavailable for inclusion. 
+The COSMOS data in Table~\ref{tab:sv_ddf_summary} therefore originate exclusively from the Small Fields Surveys (\S~\ref{sec:small_field}).
+The remaining DDFs became observable toward the end of the night beginning in July, with ELAISS1 rising first. 
+Consequently, ELAISS1 received the largest number of visits, while the other DDFs were observed less extensively due to limited SV on-sky time.
+ELAISS1 was the only field observed in all six bands, $ugrizy$. 
```

Comment:
ELAISS1 was the only field observed in all six bands, $ugrizy$, during the SV survey; COSMOS also received observations in all bands as part of the Small Fields Surveys.